File: youlexuan-manager-web/src/main/java/com/offcn/sellergoods/controller/MyUserController.java
package com.offcn.sellergoods.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.offcn.entity.PageResult;
import com.offcn.pojo.TbUser;
import com.offcn.sellergoods.service.MyUserService;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/myUser")
public class MyUserController {

    @Reference
    private MyUserService myUserService;

    /**
     * 查询+分页
     * @param user
     * @param page
     * @param rows
     * @return
     */
    @RequestMapping("/search")
    public PageResult search(@RequestBody TbUser user, int  page, int  rows){
        return myUserService.findPage(user, page, rows);
    }
}

File: youlexuan-manager-web/src/main/webapp/js/controller/myUserController.js
app.controller('myUserController' ,function($scope,$controller,myUserService) {

    $controller('baseController', {$scope: $scope});//继承

    $scope.searchEntity={};//定义搜索对象
    $scope.search = function (page,rows) {
        myUserService.search(page,rows,$scope.searchEntity).success(function (resp) {
            $scope.paginationConf.totalItems=resp.total;//总记录数
            $scope.list=resp.rows;//给列表变量赋值
        })
    }

})